.github/workflows/integration-tests-dev.yml

```diff
@@ -35,18 +35,9 @@ jobs:
           node-version: '18'
           cache: 'pnpm'
 
-      - uses: actions/cache@v3
```

Comment:
@vvo should we fix the playwright version instead like in https://github.com/vercel/storage/pull/179/files#diff-2e5b024378c3660c9f07ef3a5519710987316e2542186019c41e35a58fda20e1L50 ?

Reply:
@correttojs See https://github.com/vercel/storage/pull/183/commits/a16bc0f019b87e522c683d56062833c3803a7b0e

Reply:
and https://github.com/vercel/storage/pull/183